docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.9'
+
+services:
+
+  database:
+    image: mysql:latest
+    container_name: blog_db
+    restart: always
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: blog
```

Comment:
why the db name is blog? it should be todo

Reply:
it was just an example to practice. it was not with todo app.